client/src/components/header/Header-nav/HeaderNav.module.scss

```diff
@@ -0,0 +1,42 @@
+.headerIconsContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.headerIcon {
+  cursor: pointer;
+}
+
+.headerButton {
```

Comment:
ex: change to: 

button.headerButton {     
  margin: 15px 40px 15px 15px; 
  background-color: #fff;
}